data/json/recipes/recipe_food.json

```
@@ -416,6 +443,7 @@
     "autolearn": true,
     "qualities": [ { "id": "CUT", "level": 1 }, { "id": "COOK", "level": 2 } ],
```

```suggestion
    "qualities": [ { "id": "COOK", "level": 2 } ],
```
Cooking oil from seeds doesn't use knives.